docs/04-operation-guides/operations/evnt-01-enable-kyma-with-jetstream.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Enable Kyma with JetStream
+---
+
+This guide shows how to enable JetStream and how it ensures `at least once` delivery.
+
+### Enable JetStream
+
+Install Kyma and enable the JetStream flag by running:
+
+```
+kyma deploy --value global.jetstream.enabled=true --value global.jetstream.storage=file
+```
+
+> **NOTE:** The storage flag set to `file` enables persistence of messages and streams, if the NATS server restarts.
+
+### At least once delivery using JetStream backend
```

Comment:
```suggestion
### `At least once` delivery using JetStream backend
```

Reply:
@grego952 Do I need the highlight for the heading?

Reply:
Let me double check

Reply:
No, we don't need